recipes/pygpu/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "0.6.5" %}
+
+package:
+  name: pygpu
+  version: {{ version }}
+
+source:
+  fn: libgpuarray-{{ version }}.tar.gz
+  url: https://github.com/Theano/libgpuarray/archive/v{{ version }}.tar.gz
+  sha256: 29f44957a826083954d01de7009083fec4acb904bc51b48d3c667ef52c1f2c0b
+
+build:
+  number: 0
+  script:
+    - set INCLUDE=%LIBRARY_INC%;%INCLUDE%                            # [win]
+    - set LIB=%LIBRARY_LIB%;%LIB%                                    # [win]
+    - export CFLAGS=${CFLAGS}" -I${PREFIX}/include -L${PREFIX}/lib"  # [unix]
+    #
+    # Uncomment on next release.
+    #
+    # - python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
As noted in the comment, with the next release of `libgpuarray`, this will need to be uncommented.